assets/js/components/user-input/UserInputPreviewGroup.js

```diff
@@ -216,6 +214,79 @@ export default function UserInputPreviewGroup( {
 		toggleEditMode();
 	}, [ isScreenLoading, resetUserInputSettings, toggleEditMode ] );
 
+	function PreviewAnswers() {
```

Comment:
@ankitrox Rather than inline functions these should be components in their own files, either within the folder `assets/js/components/user-input/` or you could rename `assets/js/components/user-input/UserInputPreviewGroup.js` to `assets/js/components/user-input/UserInputPreviewGroup/index.js` and add them in the new `assets/js/components/user-input/UserInputPreviewGroup` folder.